h2o-py/h2o/grid/grid_search.py

```diff
@@ -354,6 +354,22 @@ def train(self, x=None, y=None, training_frame=None, offset_column=None, fold_co
         parms["x"] = x
         self.build_model(parms)
 
+    def resume(self, recovery_dir=None):
```

Comment:
out of scope of this PR, this PR just aims to make grid resumable without the need to provide any additional info on resume i.e. the method `resume()`, for user triggered resume it is necessary to indicate whether the resumed grid should also be recoverable since the circumstances might have changed since the last attempt